content/dns-lookup-api.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "DNS lookup API"
+date: 2018-03-29T14:49:22+02:00
+---
+
+An API that allows a WebExtension to lookup for arbitrary DNS records in highly
+efficient manner.
```

Comment:
> An API that allows a WebExtension to lookup for arbitrary DNS records in highly efficient manner.

This is a tricky sentence, as you'd normal _allow a WebExtension to **look up** a **thing**_, without the ~~for~~ in there, but for the special case of dns _lookups_  you'd say something more specific like

_An API that allows a WebExtension to perform DNS lookups for arbitrary record types in a highly efficient manner._